APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md

```diff
@@ -0,0 +1,153 @@
+- PR [#331](https://github.com/camaraproject/WorkingGroups/pull/331) ongoing. This is the first step of all the actions identified in [#100](https://github.com/camaraproject/Governance/issues/100). 
+
+## Discussion
+
+| **Agenda  Item**  |  Owner| Description |
+| ----------- | --- | ----------- |
+| **Recent updates & recap** | | - Minutes of last API backlog WG call available [here](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/MeetingMinutes/MOM-2023-10-19.md) <br> - Minutes of last TSC call available [here](https://github.com/camaraproject/Governance/blob/main/documentation/MeetingMinutes/TSC/TSC-2023-10-19-Meeting-Minutes.md). <br> - Device Swap Sub-Project created. <br> - IMEI Fraud can get started in [DeviceIdentifier](https://github.com/camaraproject/DeviceIdentifier) Sub-Project. <br> - Five new APIs were submitted into backlog, all of them coming from Open Gateway Drop #3. Three of them included in today's agenda.| 
+| **Current issues** |  |  |
+| [#271](https://github.com/camaraproject/WorkingGroups/issues/271)  | China Telecom, Huawei| **New API proposal: Site to cloud (S2C) VPN**</br> Updates: <br> - Proposed for drop #3, but unfortunately didn't go shortlisted. </br> - Progress offline among interested parties (API owners, AT&T) with API backlog and TSC mediating. Good discussion and clarification on technical and product scope. <br> - TSC reported offline that they are in position of accepting the proposal, though encourage AT&T and China Telecom aligment within the Sub-Project. <br> - Eric (Vodafone): he reported that TSC gives green light to this Sub-Project, with this decision formally made on 2nd Nov call. <br> - Jose (Telefonica): he asks Pierre (AT&T) whether AT&T would join the Sub-Project, taking guidance from TSC. <br> - Pierre (AT&T): need to align internally with the AT&T colleagues who led the Catalyst, and would come back with a response (AP 20231102-02). <br> - Once the sub-project is approved, the supporters need to provide initial maintainers (AP 20231102-02).|
+| [#294](https://github.com/camaraproject/WorkingGroups/issues/294) | Verizon| **New API proposal: Network Insights** <br> Updates: <br> - TSC asked specific and understandable API name for Network Insights. Offline discussion with GSMA and Verizon for the new API name for Network Insights. <br> - Verizon: What we have not decided yet is the name for the sub project which will have the specific contribution(yaml) tagged to. i was considering that once the project is kicked off and we have the project call we can finalize the name among the participants interested in this project. If this is holding up the whole process then lets please go ahead with "Connectivity Insights". <br> - API backlog WG will inform TSC that "Connectivity Insights" is proposed as new Sub-Project name (AP 20231019-04 cannot be closed, until TSC ok). <br> - for Sub-Project initial maintainers (AP 20231019-04 can be closed).
```

Comment:
Not explicitly discussed within TSC, but as there was no objection, I asked @MarkusKuemmerle to create the new repository as requested within this [message](https://lists.camaraproject.org/g/tsc/message/62).

Reply:
I will wait for the official confirmation from TSC to close this action point, for full transparency